ITensorGPU/test/Project.toml

```diff
@@ -1,5 +1,6 @@
 [deps]
 CUDA = "052768ef-5323-5732-b1bb-66c8b64840ba"
+cuTENSOR = "011b41b2-24ef-40a8-b3eb-fa098493e9e1"
```

Comment:
Do we need to add this as an explicit test dependency?

Reply:
I am verifying that now on my machine. I don't think we do